ci-operator/config/stackrox/stackrox/stackrox-stackrox-master.yaml

```diff
@@ -151,6 +151,12 @@ tests:
   steps:
     test:
     - ref: stackrox-initial
+- as: merge-gke-operator-e2e-tests
```

Comment:
`ci-operator/config/stackrox/stackrox/stackrox-stackrox-release-x.y.yaml` file doesn't have any mention of `*gke-operator-e2e-tests`. By some earlier agreement, we should maintain these files to match `master`. See Pull Request Checklist in https://docs.engineering.redhat.com/pages/viewpage.action?spaceKey=StackRox&title=Making+changes+to+OpenShift+CI#MakingchangestoOpenShiftCI-PullRequestchecklist

Reply:
It's not in any of the release-* files because none of these releases have [the commit](https://github.com/stackrox/stackrox/commit/8d8b9f1e3ef2bf6a570445cb667b3ba5666e0557) necessary for this job to pass. It's not a trivial change, so I don't think we should be backporting it?

I would like to challenge this agreement. It seems obvious to me that if `master` contains a new feature that is not going to be backported to old releases, then the CI config for these old releases do not need the new jobs needed to test this new feature. What am I missing?

Reply:
`release-x.y`, where `x.y` is literal thing, not numbers, is a template for next releases that will be cut from `master`.
Looks like the page did not make it clear enough that `release-x.y` is a template.
I updated the page to make the "template" idea more apparent. Please let know if it makes sense.
![image](https://github.com/openshift/release/assets/537715/e8c7a543-6f4d-4a49-b575-8e19f7bf8d02)

Reply:
Done, PTAL